syntax = "proto3";
package client;

service ClientService {
  rpc CreateClient (ClientRequest) returns (ClientReply);
}

message ClientRequest {
  string name = 1;
  string email = 2;
}

message ClientReply {
  string id = 1;
  string status = 2;
}